packages/component-library/package.json

```diff
@@ -29,6 +29,7 @@
     "react": "^16.9.0"
   },
   "dependencies": {
+    "@cultureamp/kaizen-design-tokens": "^0.1.2",
```

Comment:
The tokens package needs to be a peer dependency, as we want to tell all consumers to also add tokens as a  dependency whenever they use Kaizen. This is a really good read: https://nodejs.org/es/blog/npm/peer-dependencies/

Reply:
I made a commit with the peer dependency as `0.x` (forgetting that you'd pasted some code), thinking I should restrict it to the 0 major version in case we made some breaking changes to the tokens. What do you think?

Reply:
That's a good call, definitely the right choice.